best-practices/native/react-native/common-components/buttons/button.tsx

```diff
@@ -0,0 +1,78 @@
+import React from "react";
+import { StyleSheet, TouchableOpacity, Text, Image } from "react-native";
+import PropTypes from "prop-types";
+
+interface ButtonProps {
+  onPress: Function;
+  text: string;
+  icon: any;
+  height: number;
+  width: number;
+  backgroundColor: string;
+}
```

Comment:
Missing a semicolon here.

Reply:
For some reason my linter was removing it automatically for `Interface`s. Weird...I put it back in. Good catch!